contents/blog/product-360.md

```diff
@@ -0,0 +1,126 @@
+
+### 3. We structured the company for autonomy
+
+PostHog uses [small teams](../handbook/people/team-structure/why-small-teams), inspired by Twilio, who were inspired by Amazon. [Jeff Lawson's book](https://www.askyourdeveloper.com/) is a good 101 on this topic. TL;DR - we're 30 people, split into teams of no more than 6. The most crucial concept is that a team can ship into production without any dependencies. This optimizes for autonomy, not control.
+
+These teams are supported by product people. That means engineers are not told in detail what to do, or what is ok to merge. We put our product people into specific small teams where there is lots of help needed, but many teams run fine without them doing more than informal support (at our current scale at least).
+
+### 4. We created two (very) basic frameworks
+
+We created a Nail X framework. You tell the company "Nail X" where X is a thing you must nail. I'm not joking. [Unnuanced communication can be powerful](https://twitter.com/danluu/status/1487228574608211969). Product people decide what we need to nail.
+
+"Nail Funnels" was the first thing we did when Marcus (VP Product) joined last year. And we did. We made [Funnels](/product/funnels) in PostHog awesome.
+
+Get X reference customers was another useful framework. This came out of a board meeting. We realized that we had product market fit with our free open-source product, but we'd not yet worked on it for our PostHog Scale (paid product). We defined these as "genuinely delighted customers, using the product a lot, paying full price". We listed a few customers out we felt could achieve these three things to become a reference. Then we told the company to drop everything if needed to get it right with this group. Our product team made sure everyone was on track to achieve this.
+
+### 5. We wrote stuff down
+
+This matters to us as it prevents "information silos". I tried hard to find a less corporate way to describe that succinctly, but couldn't.
+
+_Everyone_ must work like this. This is what the exec team at PostHog does:
+
+* We share our exec meetings publicly in the company. Anyone can comment / read.
+* We write out strategy changes into a pull request. Anyone can comment / read / suggest changes.
+* We share our board meetings publicly in the company. Anyonen can comment / read.
+* We share realtime updates around fundraising. Not after the fact.
+* We have a public handbook - which is world editable
+
+### 6. Product should involve engineers
+
+Product are proactive with getting feedback. They involve engineering - many times a day, PostHog's engineering team speak to real life users. That may be over Slack, or it may be on a call.
+
+When engineers release new features, we'll encourage them to [collaborate directly with users](https://neilkakkar.com/How-I-Own-Projects-as-a-Software-Engineer.html) in real life.
```

Comment:
I think it might be worth highlighting Neil's main point a bit more in this sentence.. e.g. something along the lines of....Engineers own solving problems end-to-end, product managers find and shape the biggest problems for the engineers to solve..

Reply:
agreed - added more context here and clarified it